extensions/smallrye-metrics/deployment/src/main/java/io/quarkus/smallrye/metrics/deployment/SmallRyeMetricsProcessor.java

```diff
@@ -103,6 +103,15 @@
         @ConfigItem(name = "extensions.enabled", defaultValue = "true")
         public boolean extensionsEnabled;
 
+        /**
+         * Apply Micrometer compatibility mode, where instead of regular 'base' and 'vendor' metrics,
+         * Quarkus exposes the same 'jvm' metrics that Micrometer does. Application metrics are unaffected by this mode.
```

Comment:
It's seems that they are affected

Reply:
I don't know what you mean here. Is it about the `application_` prefix being there? That's the way it is normally (without the compatibility mode)